static/js/containers/ProfileFormContainer.js

```diff
@@ -177,6 +113,24 @@ class ProfileFormContainer extends React.Component {
     }
   }
 
+  simpleActionHelpers: Function = (): ActionHelpers => {
+    const { dispatch } = this.props;
+    return createSimpleActionHelpers(dispatch, [
+      [setWorkDialogVisibility, "setWorkDialogVisibility"],
+      [setWorkDialogIndex, "setWorkDialogIndex"],
+      [clearProfileEdit, "clearProfileEdit"],
+      [setEducationDialogVisibility, "setEducationDialogVisibility"],
+      [setEducationDialogIndex, "setEducationDialogIndex"],
+      [setEducationDegreeLevel, "setEducationDegreeLevel"],
+      [setUserPageDialogVisibility, "setUserPageDialogVisibility"],
+      [setShowEducationDeleteDialog, "setShowEducationDeleteDialog"],
+      [setShowWorkDeleteDialog, "setShowWorkDeleteDialog"],
+      [setDeletionIndex, "setDeletionIndex"],
+      [setShowWorkDeleteAllDialog, "setShowWorkDeleteAllDialog"],
+      [setShowEducationDeleteAllDialog, "setShowEducationDeleteAllDialog"],
```

Comment:
when I was initially writing this I ran into some issue with `func.name`, can't figure out what it was now though! haha

I'll rewrite it to use that.